bitnami/spring-cloud-dataflow/README.md

```diff
@@ -729,7 +729,7 @@ export RABBITMQ_ERLANG_COOKIE=$(kubectl get secret --namespace default dataflow-
 Upgrade your release (maintaining the version) disabling MariaDB and scaling Data Flow replicas to 0:
 
 ```console
-$ helm upgrade dataflow bitnami/spring-cloud-dataflow --version 0.7.4 \
+helm upgrade dataflow bitnami/spring-cloud-dataflow --version 0.7.4 \
```

Comment:
If we remove the dollar, then we need to change the code block language:
```suggestion
```bash
helm upgrade dataflow bitnami/spring-cloud-dataflow --version 0.7.4 \
```

Reply:
Thanks for the feedback @juan131 . I have adjusted to use `bash` for the code block language. Yep, you can tell a difference when viewing it for sure. Good catch.

Reply:
Thanks!